external-crates/move/crates/move-compiler/tests/move_check/expansion/pack_no_fields_single_block_expr.exp

```diff
@@ -1,9 +1,40 @@
+warning[W09002]: unused variable
+  ┌─ tests/move_check/expansion/pack_no_fields_single_block_expr.move:4:13
+  │
+4 │         let s = S { false };
+  │             ^ Unused local variable 's'. Consider removing or prefixing with an underscore: '_s'
+  │
+  = This warning can be suppressed with '#[allow(unused_variable)]' applied to the 'module' or module member ('const', 'fun', or 'struct')
+
+error[E04016]: too few arguments
+  ┌─ tests/move_check/expansion/pack_no_fields_single_block_expr.move:4:17
+  │
+4 │         let s = S { false };
+  │                 ^^^^^^^^^^^ Missing argument for field 'f' in '0x42::M::S'
+
 error[E01002]: unexpected token
   ┌─ tests/move_check/expansion/pack_no_fields_single_block_expr.move:4:21
   │
 4 │         let s = S { false };
-  │                     ^^^^^
-  │                     │
-  │                     Unexpected 'false'
-  │                     Expected an identifier
+  │                     ^ Expected a field expression
+
+warning[W09002]: unused variable
+  ┌─ tests/move_check/expansion/pack_no_fields_single_block_expr.move:5:13
+  │
+5 │         let s = S { 0 };
+  │             ^ Unused local variable 's'. Consider removing or prefixing with an underscore: '_s'
+  │
+  = This warning can be suppressed with '#[allow(unused_variable)]' applied to the 'module' or module member ('const', 'fun', or 'struct')
+
+error[E04016]: too few arguments
+  ┌─ tests/move_check/expansion/pack_no_fields_single_block_expr.move:5:17
+  │
+5 │         let s = S { 0 };
+  │                 ^^^^^^^ Missing argument for field 'f' in '0x42::M::S'
```

Comment:
Another situation where I wonder about trying to suppress subsequent errors due to bad parser.